src/authService.js

```diff
@@ -7,9 +7,35 @@ import {BaseConfig} from './baseConfig';
 
 @inject(Authentication, BaseConfig)
 export class AuthService {
+  /**
+   * The Authetnication instance that handles the token
+   * @type {Authentication}
+   */
+  authentication;
+
+  /**
+   * The Config instance that contains the current configuration setting
+   * @type {Config}
+   */
+  config;
+
   constructor(authentication, config) {
     this.authentication = authentication;
     this.config         = config;
+
+    // get token stored in previous format over
+    const oldStorageKey = config.tokenPrefix
+                        ? config.tokenPrefix + '_' + config.tokenName
+                        : config.tokenName;
+    const oldToken = authentication.storage.get(oldStorageKey);
+
+    if (oldToken) {
```

Comment:
i have a test for this in actually. (and had it tested in my app) so, i'm quite certain